core/src/main/java/feign/Param.java

```diff
@@ -44,19 +46,10 @@
    */
   boolean encoded() default false;
 
-  interface Expander {
-
-    /**
-     * Expands the value into a string. Does not accept or return null.
-     */
-    String expand(Object value);
-  }
-
-  final class ToStringExpander implements Expander {
```

Comment:
This breaks backward compatibility. If anyone specified `feign.Param.ToStringExpander` (unlikely, but it is `public` so certainly possible) anywhere, that will no longer work.

Reply:
Oh! I did not notice that it is public. I have fix it. -_-!